dspace-api/src/main/java/org/dspace/core/Context.java

```diff
@@ -131,6 +131,11 @@ private void init()
         {
             // Obtain a non-auto-committing connection
             dbConnection = new DSpace().getSingletonService(DBConnection.class);
+            if(dbConnection == null)
+            {
+                log.fatal("Cannot obtain the bean the provides a database connection, " +
```

Comment:
Picky language stuff:
1.  replace the second "the" with "which".
2.  Comma splice:  replace the comma with a period and begin a new sentence below.
With those changes, or even without, +1.